.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,40 @@
+name: Bug Report
```

Comment:
This one should be here https://github.com/coveo/ui-kit

Reply:
I've already opened one for ui-kit but miss-copied. Hadn't noticed however that search-ui doesn't have an issue tab, I'm guessing issue-ing is not allowed for this repo?